src/main/java/organice/logic/commands/DoneCommand.java

```diff
@@ -16,14 +19,13 @@
     public static final String COMMAND_WORD = "done";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finish the processing of patients and donors"
-            + "Parameters: ic/PATIENT NRIC ic/DONOR NRIC \n"
-            + "Example: " + COMMAND_WORD + " ic/s4512345A ic/s7711123C";
 
-    public static final String MESSAGE_SUCCESS = "Done processing the patient and donor";
-    public static final String MESSAGE_NOT_PROCESSED = "Donor or patient NRIC must be valid";
+            + "Parameters: ic/PATIENT NRIC ic/DONOR NRIC res/[pass/fail] \n"
```

Comment:
I think you need to add a "\n" before this string.

Reply:
amended